pennylane/ops/op_math/decompositions/controlled_decompositions.py

```diff
@@ -414,14 +628,14 @@ def _ctrl_decomp_bisect_od(U, wires, skip_initial_cx=False):
     control_k2 = wires[mid:-1]
 
     if not skip_initial_cx:
-        ops.ctrl(ops.X(wires[-1]), control=control_k1, work_wires=control_k2)
+        _controlled_x(wires[-1], control_k1, control_k2, "dirty")
 
     ops.QubitUnitary(a, wires[-1])
-    ops.ctrl(ops.X(wires[-1]), control=control_k2, work_wires=control_k1)
+    _controlled_x(wires[-1], control_k2, control_k1, "dirty")
     ops.adjoint(ops.QubitUnitary(a, wires[-1]))
-    ops.ctrl(ops.X(wires[-1]), control=control_k1, work_wires=control_k2)
+    _controlled_x(wires[-1], control_k1, control_k2, "dirty")
     ops.QubitUnitary(a, wires[-1])
-    ops.ctrl(ops.X(wires[-1]), control=control_k2, work_wires=control_k1)
+    _controlled_x(wires[-1], control_k2, control_k1, "dirty")
```

Comment:
Are their arg names removed for some certain compatibility with something? I would prefer keeping the argname since when positional args more than 2 it's hard to immediately recall what is which